### ISSUE TITLE
Android App Error: Failing to load previous link ID after first 24 hours

### ISSUE THREAD

**Issue by jrswab** [keybase/client#7267]
I downloaded the Android app and it worked great for about 24 hours (maybe less) and now when logging in it give me this error message.
`Error: tried to load previous link ID XXXXXXX, but link not found`

Logging in works great online but have not tried the desktop app since I'm in the office.

I did see a similar issue here on GitHub but was not exactly the same wording/language:
[Bug 7262](https://github.com/keybase/client/issues/7262)

FWIW: I did restart the app, tired on LTE and not wifi, and rebooted my device to no avail.

If you need any more information I'm happy to provide.

**Comment by maxtaco** [comment]
Thanks @jrswab for the report. We have a fix in the latest build, we just have to prepare and push out a fix. Thanks!

**Comment by ogfomk** [comment]
I believe that this is the issue I have had. RE: [https://github.com/keybase/client/issues/7277](https://github.com/keybase/client/issues/7277). I am not sure that 24 hours is the case but the message is definitely the same. As soon as I can recreate I will upload a log file but it may not be possible as the only way to get into the app is through this process. The way to make the app useable again is to delete data and cache and then I image the log files will be gone to.

**Comment by ogfomk** [comment]
@jrswab The very first time I tried this whole Keybase thing out I started on phone. When the link ID error happened I was in a heap of trouble so I installed the Linux version since I could log on via the web. That is how I was able to recover. I am having quite a learning curve here as I am PGP/Encrytion challenged. 

That being said, I am trying to replicate the problem so that I can upload some photos and show what I saw. There was also some shell text too!  I could not make a screen shot due to app security... Now I am thinking maybe there should be a development mode that allows this? Also maybe a way to send log files when you can not log back in?

**Comment by jrswab** [comment]
@ogfomk I don't know as much about PGP as I should. Most of my encryption knowledges comes from my intense 6 month study of bitcoin about two years ago. My motherboard fried and now I have to pull my hard drive out to get my 'paper' key. (should have printed that out or something lol) I love security and being apart of beta tests so this is all right up my alley hahaha

I did seem some text in a shell like fashion myself once or twice. That being said it sounds like they know what is happening on the app and it will be fixed soon enough.

**Comment by maxtaco** [comment]
hey guys, sorry about this, known bug, fixed in the next release.  definitely nuking your cache is a workaround.

cc: @oconnor663  and @malgorithms

**Comment by jrswab** [comment]
Thanks @maxtaco ! I tried nuking cache and re-opening but I still got the error until I deleted data. However, I did not restart my device after wiping the app's cache.

**Comment by ogfomk** [comment]
@jrswab Yes I should have been more specific you have to delete your data.  Kind of like in Linux when you have an new program and the whole UI gets hosed because you did something that you have no idea what you did and you can not find out how to reverse... You just delete the hidden files associated with the application and restart back to square one. 

@maxtaco This is great! I am glad that my testing can help to improve this whole ecosystem. I'll try to break it more! I'm imagining that the futurism of this is that you have a public verifiable ID and you have the ability to control how you are communicated with. Duh I know it is nothing new but you all seem to have pulled it all together quite nicely. I want to contribute as much as I can.

**Comment by ogfomk** [comment]
I have uploaded some photos of this issue. It happen again. There is no way to login and send Feedback regarding these issues. https://plus.google.com/+AlexanderNuttall/posts/4ap8fqXCpM2

**Comment by ogfomk** [comment]
Four Errors of the Same Issue
1. There was an error provisioning
2. Can't load inbox shell
3. Error: Tried to y link ID...
4. Can't load inbox

**Comment by ogfomk** [comment]
I was given notification on my Andriod app this morning that someone @jrswab had sent me a message so something is working but I still could not log in to open the app for use.

1. At the top in the UI in black: "Can't load inbox."
2. Also circle with exclamation point in the upper left corner.
3. Under "Passphrase" the message:" Error: tried to load link ID (bunch hex numbers) but link not found."
4. If you touch the top black banner this exposes a shell with "Can't load inbox" at the top then of about 25 addresses with the words at the bottom: [native code]

Same as the post I made before https://plus.google.com/+AlexanderNuttall/posts/4ap8fqXCpM2

But something is working because the app told me that someone contacted me with a notice.

**Comment by jrswab** [comment]
Just got the update on my device and login works without a hitch! Thank you so much for being so open with this issue!

**Comment by maxtaco** [comment]
Great news, thanks for your patience!

**Comment by ogfomk** [comment]
I added my phone again. @jrswab was smart and just waited. I'll test it for 24 plus hours. it should let me know and others.  Still have issue with camera not having permission to be used by app, but that is another issue. Thanks @maxtaco I like what is happening and I think it can be sustainable and very useful.

**Comment by jrswab** [comment]
@ogfomk have you tried going into settings -> apps -> KeyBase -> Permissions and checking off the camera option? I found this was off by default.

**Comment by ogfomk** [comment]
@jrswab By default the Keybase app has the permission "take pictures and videos." As far as accessing this through the app itself you can not do this until you have established your account or added the device. The process to add the device involves using the camera or typing in the code. So you can't get to the app settings until you have established you have an account or have started a new one. On android apps by default the app has permission to use the camera. https://goo.gl/photos/8ArrtiJD8KQmzjzG6
